internal/report/report.go

```diff
@@ -241,15 +242,27 @@ func (rpt *Report) newGraph(nodes graph.NodeSet) *graph.Graph {
 	for _, f := range prof.Function {
 		f.Filename = trimPath(f.Filename)
 	}
-	// Remove numeric tags not recognized by pprof.
+	// Removes all numeric tags except for the bytes tag prior
+	// to making graph.
+	// TODO : modify to select first numeric tag if no bytes tag
```

Comment:
Nit: remove the space before the colon make the text a proper sentence. I.e. "TODO: Do something."